pytest/tests/shardnet/scripts/create_account.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
ok - now I see, 

so this tool is used to create the validator account (for the keys that are already present on the machine) - and you have to pass the private&public key for 'near' user as arguments.

optional: I'd probably use node.machine.upload to upload the near.json to the machines instead.